docs/Monitored_Only_ZFS.md

```diff
@@ -0,0 +1,208 @@
+
+3. Install epel-release and configure necessary repos on each virtual machine:
+    ```
+    vagrant sh -c '\
+    sudo yum -y install epel-release && \
+    sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo && \
+    sudo yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/ && \
+    sudo ed <<EOF /etc/yum.repos.d/mirror.centos.org_centos_7_extras_x86_64_.repo
+    /enabled/a
+    gpgcheck=1
+    gpgkey=http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7
+    .
+    wq
+    EOF
+    '
+    ```
+
+   ```
+   vagrant sh -c '\
+   sudo yum-config-manager --add-repo https://build.whamcloud.com/lustre-b2_10_last_successful_server/ && \
+   sudo sed -i -e "1d" -e "2s/^.*$/[lustre]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/build.whamcloud.com_lustre-b2_10_last_successful_server_.repo && \
+   sudo yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/ && \
+   sudo sed -i -e "1d" -e "2s/^.*$/[e2fsprogs]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/downloads.hpdd.intel.com_public_e2fsprogs_latest_el7_.repo
+   ' mds1 mds2 oss1 oss2
+   ```
+
+
+4.  Create zfs installer and install the zfs packages on the following servers: mds1, mds2, oss1, and oss2
+   ```
+       cd ~/downloads
+       tar xzvf iml-4.0.0.0.tar.gz
+       cd iml-4.0.0.0
+       ./create_installer zfs
+       for i in {2200..2203}; do scp -P $i ~/Downloads/iml-4.0.0.0/lustre-zfs-el7-installer.tar.gz vagrant@127.0.0.1:/tmp/.; done
+       # password is "vagrant"
+       vagrant sh -c 'cd /tmp; sudo tar xzvf lustre-zfs-el7-installer.tar.gz; cd lustre-zfs; sudo ./install' mds1 mds2 oss1 oss2
+   ```
+
+
+## Installing IML:
+
+1.  Copy the IML build to the /tmp directory in your admin node:
+    ```
+    scp -P 2222 ~/Downloads/iml-4.0.0.0.tar.gz vagrant@127.0.0.1:/tmp/.
+    # password is "vagrant"
+    ```
+2. ssh into the admin box and install the build:
+    ```
+    vagrant ssh
+    [vagrant@ct7-adm ~]$ sudo su - # (or "sudo -s")
+    [vagrant@ct7-adm ~]# cd /tmp
+    [vagrant@ct7-adm ~]# tar xvf <buildname>.tar.gz
+    [vagrant@ct7-adm ~]# cd <build folder>
+    [vagrant@ct7-adm ~]# ./install --no-dbspace-check
+    ```
+3. Update the /etc/hosts file on your computer to include the following line:
+    ```
+    127.0.0.1 ct7-adm.lfs.local
+    ```
+4. Test that a connection can be made to IML by going to the following link in your browser:
+https://ct7-adm.lfs.local:8443
+
+## Adding and Configuring Servers
+You should now be able to see IML when navigating to https://ct7-adm.lfs.local:8443. Click on the login link at the top right and log in as the admin. Next, go to the server configuration page and add the following servers:
+```
+ct7-mds[1,2].lfs.local,ct7-oss[1,2].lfs.local
```

Comment:
Drop `ct7-` prefixes.

But again. this feels like a significant duplication of other text.

Reply:
Changed. Perhaps this should just reference the link to the other text.